### PR TITLE
chore(local-deck): change webpack.dev configuration

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**
In deck.gl there are a lot of typescript issues in multiple files.
Current project setup can't handle such issues in `yarn start-local-deck` mode.
To prevent such issues just disable `ts-loader` for now.

### CODE REVIEW

File: webpack.dev.config.js [hunk 1]
```diff
@@ -51,6 +51,30 @@ function getAliasesForLocalDependencies(env) {
...
 }
 
+/**
+ * Returns loader for 'ts' and 'tsx' files.
+ * If we are using '--deck' env variable we should avoid TypeScript issues from deck.gl folder.
+ * TODO: We should always use ts-loader once all issues with deck.gl types have been resolved.
+ */
+function getLoaders(env) {
+  // Use simple babel loader without typescript to avoid local deck.gl typing issues.
+  if (env["deck"]) {
```
Comment (belom88) [inline webpack.dev.config.js:61]: I had the issue with "loaders" as well. When I try to add new options to Tileset3D there type errors occur.

**Review by belom88 — CHANGES_REQUESTED** [review]
I'm not sure we need these changes. Despite errors the application starts and works after I close overlapping panel with errors.
image
I guess it is sufficient for development.

File: webpack.dev.config.js (hunk 1 above)
Comment (dryabinin-actionengine) [reply webpack.dev.config.js:61]: > I'm not sure we need these changes. Despite errors the application starts and works after I close overlapping panel with errors. image I guess it is sufficient for development.

If you will continue use applications with these errors and try to add some changes to deck.gl locally application won't update with these changes.

Comment (dryabinin-actionengine) [reply webpack.dev.config.js:61]: Also I think we should do the same changes for loaders in dev webpack config.

**Review by belom88 — APPROVED** [review]

**Comment by dryabinin-actionengine** [comment]
> 

Good catch! Thank you!

**Review by belom88 — APPROVED** [review]

**Pull request merged**